app/views/auth/registrations/edit.html.haml

```diff
@@ -23,6 +23,8 @@
 
     .actions
       = f.button :button, t('generic.save_changes'), type: :submit, class: 'button', disabled: current_account.suspended?
+  - elsif omniauth_only?
+    = link_to t('users.go_to_sso_account_settings'), sso_account_settings
```

Comment:
Maybe check that it's defined?

```suggestion
  - elsif omniauth_only? && sso_account_settings.present?
    = link_to t('users.go_to_sso_account_settings'), sso_account_settings
```

Reply:
On this note: I wanted to ask whether this option should also be triggered when seamless_external_login (aka. LDAP) is activated?